mixpanel_query/client.py

```diff
@@ -33,11 +35,12 @@ class MixpanelQueryClient(object):
     DATA_TYPE_UNIQUE = 'unique'
     VALID_DATA_TYPES = (DATA_TYPE_GENERAL, DATA_TYPE_AVERAGE, DATA_TYPE_UNIQUE)
 
-    def __init__(self, api_key, api_secret, timeout=None):
+    def __init__(self, api_key, api_secret, auth_class=SignatureAuth, timeout=None):
```

Comment:
If one of the goals of this PR is to _not_ affect existing users, we may want to add this new keyword arg to the last position in this `__init__` signature.

There _may_ be some users who are using positional args to invoke this method like so:

```python
mpqc = MixpanelQueryClient(key, secret, 600)
```

The call above "breaks" if the user upgrades.